web/gds-user-ui/src/modules/dashboard/organization/useOrganizationListQuery.tsx

```diff
@@ -13,7 +13,7 @@ export function useOrganizationListQuery(): OrganizationQuery {
 
   return {
     getAllOrganizations: query.refetch,
-    organizations: query.data?.data as Organization[],
+    organizations: query.data?.data as any,
```

Comment:
I think it is a bad idea to add `any` here because it could make debugging harder. What if you create a type based on what you get from backend ?